packages/api-explorer/src/Doc.jsx

```diff
@@ -105,6 +106,9 @@ class Doc extends React.Component {
   }
 
   onSubmit() {
+    if(this.anyOfSwitcher) {
+      this.anyOfSwitcher.dispatchEvent(new Event('change'))
```

Comment:
Why?

Reply:
The event is listened by the custom editor in order to trigger updates, the change event is actually dispatched by jsoneditor, here it seems like we're abusing it for getting formsubmission

Reply:
I've modified the listener interface by using an observable pattern, the custom editor implements the `onFormSubmission` function that gets invoked by the Doc component.

This way we can avoid performing the request if the scheme is not set, as of now the request was invoked even if the schema error was shown